src/algebra/ring/basic.lean

```diff
@@ -1194,12 +1220,28 @@ lemma mul_self_eq_one_iff [non_assoc_ring R] [no_zero_divisors R] {a : R} :
   a * a = 1 ↔ a = 1 ∨ a = -1 :=
 by rw [←(commute.one_right a).mul_self_eq_mul_self_iff, mul_one]
 
+namespace units
+
+@[field_simps] lemma divp_add_divp [comm_ring α] (a b : α) (u₁ u₂ : αˣ) :
```

Comment:
Good point, added a test case for that, too. 

(Note that in practice this priority isn't too important since `field_simp, ring,` solves the goal regardless of which lemma takes priority.)